modules/mod_tags_popular/src/Helper/TagsPopularHelper.php

```diff
@@ -38,7 +38,7 @@ public static function getList(&$params)
 		$user        = Factory::getUser();
 		$groups      = $user->getAuthorisedViewLevels();
 		$timeframe   = $params->get('timeframe', 'alltime');
-		$maximum     = $params->get('maximum', 5);
+		$maximum     =(int) $params->get('maximum', 5);
```

Comment:
```suggestion
		$maximum     = (int) $params->get('maximum', 5);
```
Please fix this code style error. Thanks in advance.